assets/app/views/directives/osc-key-values.html

```diff
@@ -23,7 +25,15 @@
                  spellcheck="false"
                  osc-input-validator="value">
         </div>
-        <button class="btn btn-default" ng-click="addEntry()" ng-disabled="form.key.$error.oscKeyValid || form.value.$error.oscValueValid">Add</button>
+        <button
+          class="btn btn-default"
+          ng-click="addEntry()"
+          ng-disabled="form.key.$error.oscKeyValid || form.value.$error.oscValueValid || form.key.$error.oscUnique">
```

Comment:
That would be nice.  This is a lot of items in `ng-disabled`, looks ugly.
I'll check it.